### PR TITLE
(1571) Remove accountable_organisation fields

### PR DESCRIPTION
**Pull request by leeky**
These fields can be inferred from `extending_organisation`, so they don't need to be stored separately.

## Next steps

- [ ] Is an ADR required? An ADR should be added if this PR introduces a change to the architecture.
- [ ] Is a changelog entry required? An entry should always be made in `CHANGELOG.md`, unless this PR is a small tweak which has no impact outside the development team.
- [ ] Do any environment variables need amending or adding?
- [ ] Have any changes to the XML been checked with the IATI validator? See [XML Validation](https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/blob/develop/doc/xml-validation.md)

### CODE REVIEW

File: app/models/activity.rb [hunk 1]
```diff
@@ -346,6 +340,36 @@ def funding_organisation
     service_owner
   end
 
+  def accountable_organisation
+    return service_owner if fund? || programme?
+
+    extending_organisation.is_government? ? service_owner : extending_organisation
+  end
+
+  def accountable_organisation_name
```
Comment (CristinaRO) [inline app/models/activity.rb:349]: These getters aren't being tested right now, is that ok?

**Review by CristinaRO — COMMENTED** [review]
One question re: the code. Also, did you intentionally not describe this change in the changelog?

**Comment by leeky** [comment]
@CristinaRO I usually only add a changelog entry for features that impact the end user, as per the instructions on the checklist itself.

File: app/models/activity.rb (hunk 1 above)
Comment (leeky) [reply app/models/activity.rb:349]: Good point! I've pushed an update which adds a spec for each of the getters

**Comment by CristinaRO** [comment]
> @CristinaRO I usually only add a changelog entry for features that impact the end user, as per the instructions on the checklist itself.

Eh, I find that guideline to be highly subjective. I tend to err on the left side of the equation ("an entry should always be made"), because what is "a small tweak which has no impact outside the development team" is fuzzy to me.

**Review by CristinaRO — APPROVED** [review]
Thanks for the clear commit messages and making the tweaks!

**Pull request merged**